hail_search/queries/base.py

```diff
@@ -343,6 +344,9 @@ def _add_project_ht(self, families_ht, project_ht, default, default_1=None):
         if default_1 is None:
             default_1 = default
 
+        if not project_ht.any(hl.is_defined(project_ht.family_entries)):
+            return families_ht, False
```

Comment:
when would a family be included in a search but not in project?

Reply:
so this case is reached when there is data loaded for a family but there are no rows in the table that match the search after its been filtered for inheritance, quality, and location. In cases where you are doing a restrictive search in a single gene across hundreds of projects, many of them will end up not having any hits so skipping the join improves performance